Sources/Prototyper/Feedback/EditScreenshotView.swift

```diff
@@ -16,17 +15,17 @@ struct EditScreenshotView: View {
     /// Environment variable for presentationMode to dismiss the View.
     @Environment(\.presentationMode) var presentationMode
     
-    /// The State variable holds the current Drawing being drawn on this View.
-    @State var currentDrawing: Drawing?
-    /// This State variable holds all the drawings and is initialised with the existing strokes when the View appears.
-    @State var allDrawings: [Drawing] = []
-    /// The frame data of the markup image in the cuurent View.
-    @State var rect: CGRect = .zero
-    /// The initial color for markup when the View appears.
-    @State var color: Color = .primary
-    /// This attribute is updated when the user wants to change the Markup Color.
-    @State var colorPickerShown: Bool = false
+    @State var canvas = PKCanvasView()
+    @State var isDraw = true
+    @State var color: Color = .black
+    @State var type: PKInkingTool.InkType = .pencil
+    @State var colorPicker = false
+    @State var inkTypeImage: String = "pencil"
     
+    ///Changes the pencil image whether you are drawing or want to erase something
+    var pencilImage: String {
+        isDraw ?  "pencil.slash" : "pencil.and.outline"
```

Comment:
Maybe use pencil.slash and pencil as this is a more clearer change of the state?